### PR TITLE
Add API conversion interface for Dropout layer.

### PR DESCRIPTION
**Pull request by jihobak**

### CODE REVIEW

File: keras/legacy/interfaces.py [hunk 1]
```diff
@@ -74,3 +74,44 @@ def wrapper(*args, **kwargs):
...
+                                'and keyword argument for argument '
+                                '`rate` '
+                                '(`p` in the legacy interface).')
+            if 'rate' in kwargs:
+                raise_duplicate_arg_error('p', 'rate')
+            p = kwargs.pop('p')
+            args = (args[0], p)
+
+        if is_converted:
+            signature = '`p` is changed to `rate`, still positional argument'
```
Comment (fchollet) [inline keras/legacy/interfaces.py]: This should be literally what the function call should look like. So the user can just copy and paste it.

File: keras/legacy/interfaces.py [hunk 2]
```diff
@@ -74,3 +74,44 @@ def wrapper(*args, **kwargs):
...
+    """
+    @six.wraps(func)
+    def wrapper(*args, **kwargs):
+        if len(args) > 2:
+            # The first entry in `args` is `self`.
+            raise TypeError('The `Dropout` layer can have at most '
+                            'one positional argument (the `rate` argument).')
+
+        # p
+        is_converted = False
```
Comment (fchollet) [inline keras/legacy/interfaces.py]: This could simple read "converted"

File: keras/legacy/interfaces.py [hunk 3]
```diff
@@ -74,3 +74,44 @@ def wrapper(*args, **kwargs):
...
+        A constructor conversion wrapper.
+    """
+    @six.wraps(func)
+    def wrapper(*args, **kwargs):
+        if len(args) > 2:
+            # The first entry in `args` is `self`.
+            raise TypeError('The `Dropout` layer can have at most '
+                            'one positional argument (the `rate` argument).')
+
+        # p
```
Comment (fchollet) [inline keras/legacy/interfaces.py]: Unclear what purpose this line serves.

**Comment by jihobak** [comment]
Sorry, I have some mistake, I closed this PR

File: keras/legacy/interfaces.py (hunk 3 above)
Comment (jihobak) [inline keras/legacy/interfaces.py]: @fchollet  Thanks, 
 This part, as you show like Dense layer, 'Dropout' layer can also have at mosts one positional argument. That's why I keep this line.

**Pull request closed without merging**